cmd/guacgql/cmd/server.go

```diff
@@ -172,20 +174,12 @@ func validateFlags() error {
 	return nil
 }
 
-func getGraphqlServer(ctx context.Context) (*handler.Server, error) {
-	var topResolver resolvers.Resolver
-
-	backend, err := backends.Get(flags.backend, ctx, getOpts[flags.backend](ctx))
-	if err != nil {
-		return nil, fmt.Errorf("Error creating %v backend: %w", flags.backend, err)
-	}
-	topResolver = resolvers.Resolver{Backend: backend}
-
+func GetGraphqlServer(ctx context.Context, backend backends.Backend) *handler.Server {
```

Comment:
If the intention is to have this be used outside the cmd itself, can we move it to a `pkg/` path? Perhaps `pkg/assembler/server`?

Reply:
great idea! I'll refactor it into `pkg/assembler/server`

Reply:
great idea! I'll refactor it into `pkg/assembler/server`

Reply:
done!